packages/flutter/lib/src/material/date.dart

```diff
@@ -203,15 +454,16 @@ typedef SelectableDayPredicate = bool Function(DateTime day);
 ///  * [showDateRangePicker], which displays a dialog that allows the user to
 ///    select a date range.
 @immutable
-class DateTimeRange {
-  /// Creates a date range for the given start and end [DateTime].
+@optionalTypeArgs
+class DateTimeRange<T extends DateTime> {
+  /// Creates a date range for the given start and end [T].
```

Comment:
This probably doesn't work, since there's not a doc for a class called "T". Change it to something like:
```suggestion
  /// Creates a date range for the given start and end date time.
```

Reply:
Updated